src/modules/emote_menu/components/Sidebar.module.css

```diff
@@ -1,33 +1,81 @@
 .sidebar {
-  width: 52px;
+  display: flex;
+  flex-direction: column;
+  height: 349px;
+}
+
+.content {
   height: 100%;
   overflow-y: scroll;
   background-color: var(--bttv-rs-bg-well);
   color: var(--bttv-rs-text-heading);
   scrollbar-width: none;
+  height: 300px;
+  width: 48px;
 }
 
-.sidebar::-webkit-scrollbar {
+.content::-webkit-scrollbar {
   display: none;
 }
 
 .active {
-  background-color: var(--bttv-rs-bg-card);
+  background-color: var(--bttv-rs-bg-card) !important;
+  /* border-right: 2px solid var(--bttv-rs-navs-selected) !important; */
 }
 
 .navItem svg {
   margin: 0 !important;
   width: 100%;
 }
 
-.navItemContent {
+.dragging {
+  background-color: var(--bttv-rs-bg-well);
+  border-radius: 4px;
+  color: var(--bttv-rs-text-heading);
+  display: flex;
+}
+
+.navItem {
+  /* border-right: 2px solid var(--bttv-rs-divider-border); */
```

Comment:
?

Reply:
As we're no longer using rsuite's components, we're losing their styling of the purple flair against active categories. Will address this during the design pass if we should still persist their styling.